### ISSUE TITLE
How to apply minigames?

### ISSUE THREAD

**Issue by ghost** [Mrmaxmeier/BombSquad-Community-Mod-Manager#62]
I don't know how to actually apply mods. I have enabled modding, & copied installer.py. it works perfectly fine, but when I install a mod (the one I tried was Super Smash) and quit & open, nothing happened except it saying that it was downloaded. I'm trying this on a windows 10 laptop that is pretty decent.

**Comment by MattZ45986** [comment]
When you click the initial 'play' button, you should be on the screen that gives you the option 'Free-for-all,' 'Teams,' or 'Co-op.' If the mod-manager is downloaded and installed properly then there should be a button in the top-right that says 'custom...' which is the button you want to click.  Then it opens up the menu with all of the included mods and any you have already.  Let me know if you have a problem.

**Comment by rahulraman0108** [comment]
But the custom button appears after you download and install the quickGameButton.py. So @switch64 in Mod Manager window click on the 'Quick-Game Button' mod then download that mod and restart the BombSquad game. Am I wrong @MattZ45986 ?

**Comment by MattZ45986** [comment]
You are correct--that would work if he doesn't already have it.  However, if @switch64 is not able to access the mods anywhere after visibly installing them, the button alone will not solve his problem.  If he has any further difficulty, he will let us know.

**Comment by rahulraman0108** [comment]
Hmm, yeah I not thought that.
